### DISCUSSION TITLE
Support enterprise orgs with SSO login requirements

### DISCUSSION THREAD

**Discussion by s0** [community/community#15253] [Enterprise] [answered]
When navigating to cs.github.com, if you are part of an organisation that has additional requirements for you to log-in via SSO, rather than being prompted / redirected to login to SSO (as is what happens on github.com when trying to access private repos for your organisation), results for your organization just don't appear.

I think it would be good to at least display a little alert indicating that you are not fully logged-in to your organization, and results from that organization may be limited / only public results included, with a link to log-in.

Beyond this, currently there seems to be a bug where if you navigate to cs.github.com, then go to an org repo, login via SSO, and then back to cs.github.com, results continue to not appear unless you clear cookies for cs.github.com.

**Comment by colinwm** [comment, marked as answer]
Hey @s0, actually cs.github.com IS designed to work with SSO, and it should show an SSO warning on the homepage and on search result pages. E.g. here's my cs.github.com homepage (I'm currently not SSO'd to the Microsoft org):

<img width="720" alt="image" src="https://user-images.githubusercontent.com/78169714/164286049-c63377cd-7199-475f-bb20-d05cca7269fe.png">

What's the org that you're experiencing this issue with? Maybe there's something unique about it...

**Reply by s0** [reply]
hmm strange, I can't reproduce what I saw yesterday, so it may have been a transient bug / some bad state with the cookies. But my colleague definitely managed to get it into a state where neither the banner was displaying, and it was producing no results from the private repos.

WFIW the organisation was `transferwise`
